### PR TITLE
Issue #2228 - Coursebuffet: Should trigger on Learn X Online

### PR DESCRIPTION
**Pull request by Mailkov**
Issue #2228 - Coursebuffet: Should trigger on Learn X Online

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Mailkov Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by moollaza** [comment]
@Mailkov welcome and thanks for contributing! This LGTM, but I just noticed that the IA isn't actually working locally or in production. It looks like it's throwing a JS error, would you be interested in fixing that as well?

**Comment by Mailkov** [comment]
Yes, I`m interested in fixing JS error.

**Comment by Mailkov** [comment]
@moollaza I tested IA in production with query: "computer science online course" and it works ... Can you explain what exactly went wrong ?

**Comment by Mailkov** [comment]
@moollaza I think IA is now ok , I could not test on DuckPan Server because when I clicked on button the new browser tab give me a blank page.

**Comment by Mailkov** [comment]
@moollaza  I tested my changes on DuckPan Server finally, IA now works.

File: share/spice/coursebuffet/coursebuffet.js [hunk 1]
```diff
@@ -6,9 +6,9 @@
         };
 
         // Get original query.
-        var script = $('[src*="/js/spice/movie/"]')[0],
+        var script = $('[src*="/js/spice/coursebuffet/"]')[0],
```
Comment (moollaza) [inline share/spice/coursebuffet/coursebuffet.js]: :+1:  -- Thanks for fixing, I can't believe we had that in there!

**Comment by moollaza** [comment]
@Mailkov Thanks! This LGTM :+1:

Congrats on your first PR :) -- I look forward to seeing more. Hopefully @MariagraziaAlastra will bug you to keep contributing :stuck_out_tongue_winking_eye:

**Comment by MariagraziaAlastra** [comment]
@Mailkov yo, well done! :cactus:

**Pull request merged**